package.json

```diff
@@ -36,10 +36,10 @@
     "eventemitter2": "~2.2.0",
     "object-assign": "^4.0.1",
     "pngparse": "^2.0.1",
-    "ws": "^1.1.1",
+    "ws": "^3.3.1",
     "xmldom": "^0.1.19",
     "cbor-js": "^0.1.0",
-    "socket.io": "1.4.8"
+    "socket.io": "2.2.0"
```

Comment:
Travis fails due to old nodejs 4.2.6 on the build/test image that doesn't support ES6 syntax (which socket.io 2.2.0 apparently uses).